test/generateMongoidTypes.test.ts

```diff
@@ -0,0 +1,29 @@
+import MongoidTypeGenerator from "../src/generators/MongoidTypeGenerator";
+import { entityCollection } from './fixture/entityCollection.json'
+import EntityCollection from "../src/model/dataTypes/EntityCollection";
+
+describe('generateMongoidTypes', () => {
+
+  test('Should generate mongoid models successfully', async() => {
+    const collection = <EntityCollection>JSON.parse(JSON.stringify(entityCollection));
```

Comment:
Codacy found an issue: [Use 'as EntityCollection' instead of '<EntityCollection>'.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5738952)